test/multiapi/packages/multiapi-test-2017-10-01/package-lock.json

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
do you need to check in these test package-lock.json files? It seems like you could tweak your .gitignore to ignore all lockfiles under test?

Reply:
This is actually the reason why the `ms-rest-js` repo's build is failing: `npm ci` is being used by Lerna when building this test folder and since the `package-lock.json` isn't there the install command fails.